pom.xml

```diff
@@ -144,14 +144,6 @@
                 <version>${project.version}</version>
             </dependency>
 
-            <!-- PROTOBUF API -->
```

Comment:
Why did you remove these lines?

Reply:
They are a duplicate of Line 128-133, but in difference to 128-133, JDK8 is hardcoded. 
It lead to several maven issues.